pipeline/packer/scr-web-service.json

```diff
@@ -0,0 +1,75 @@
+{
+  "_description": "This template produces a docker container for the SCR web service and pushes the result to the reository.",
+  "variables": {
+    "app_dir": "/usr/src/app",
+    "common_dir": "common",
+    "scr_dir": "SCR",
+    "scr_service_dir": "SCRWebService",
+    "repository": "{{env `SCR_REPOSITORY`}}",
+    "tag": "{{env `BUILD_TAG`}}",
+    "docker_registry_url": "{{env `SCR_REPOSITORY_URL`}}"
+  },
+  "builders": [
+    {
+      "type": "docker",
+      "image": "python:3-slim",
+      "commit": true,
+      "changes": [
+        "EXPOSE 9000",
```

Comment:
again, port binding on 9000 seems random.

likelihood being port 80 will want to be used and exposed from the docker containers.